app/views/incomeSources/add/CheckBusinessDetails.scala.html

```diff
@@ -154,15 +155,19 @@
                     </a>
                 </dd>
             </div>
-            @if(viewModel.businessAccountingMethod.nonEmpty) {
+            @if(viewModel.skippedAccountingMethod) {
                 <div class="govuk-summary-list__row">
                     <dt class="govuk-summary-list__key">@getMessage("accounting-method")</dt>
-                    <dd class="govuk-summary-list__value" id="business-accounting-value">@{
-                        viewModel.businessAccountingMethod.getOrElse("")
-                    }</dd>
+                    <dd class="govuk-summary-list__value" id="business-accounting-value">
+                        @if(viewModel.businessAccountingMethod.getOrElse("") == "cash"){
```

Comment:
not sure if this if/else statement is correct,
ie. if viewModel.businessAccountingMethod is None ?

Reply:
updated to match case